tests/unit/core/server/testServer.js

```diff
@@ -23,33 +33,50 @@ describe('core/server/base', () => {
     global.webpackIsomorphicTools = {
       assets: () => fakeAssets,
     };
+    mockUserApi = sinon.mock(userApi);
   });
 
   afterEach(() => {
-    Helmet.canUseDOM = _helmentCanUseDOM;
+    Helmet.canUseDOM = _helmetCanUseDOM;
     delete global.webpackIsomorphicTools;
   });
 
-  function testClient({ stubRoutes = defaultStubRoutes } = {}) {
-    function createStoreAndSagas() {
-      const sagaMiddleware = createSagaMiddleware();
-      return {
-        store: createStore(
-          combineReducers({ reduxAsyncConnect }),
-          // Do not define an initial state.
-          undefined,
-          applyMiddleware(sagaMiddleware),
-        ),
-        sagaMiddleware,
-      };
+  function createStoreAndSagas({
+    reducers = { reduxAsyncConnect },
```

Comment:
I think this should default to:

```js
{ reduxAsyncConnect, api: apiReducer, user: userReducer }
```

because really the server requires all of those to operate. I don't see a strong case for customizing the reducer list in any of the tests below.

Reply:
great, it simplifies the test cases.